cmd/convox/start.go

```diff
@@ -86,7 +96,7 @@ func cmdStart(c *cli.Context) error {
 	cache := !c.Bool("no-cache")
 	sync := !c.Bool("no-sync")
 
-	r := m.Run(dir, app, cache, sync)
+	r := m.Run(targetService, targetCommand, dir, app, cache, sync)
```

Comment:
we could simply do run and add service to the run options, that way it work exactly as it does now and we wouldn't need to broaden the manifest API